examples/observers/async_openai_example.py

```diff
@@ -0,0 +1,27 @@
+import asyncio
```

Comment:
Opened this issue for smoothening test scripts https://github.com/cfahlgren1/observers/issues/58